src/app/screens/BoardsList/layout.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import { ROUTES } from '../../../config/routes';
+import { propTypes as boardPropTypes } from '../../../redux/boards/reducer';
+import './styles.css';
+
+const BoardsList = ({ boards, onDeleteBoard }) => (
+  <ul>
+    {boards.map(board => (
+      <div className="board-card" key={board.id}>
+        <Link to={ROUTES.BOARD_DETAIL(board.id)}>
+          <h2>{board.name}</h2>
+        </Link>
+        <button onClick={() => onDeleteBoard(board.id)}>X</button>
+      </div>
+    ))}
+  </ul>
+);
+
+BoardsList.propTypes = {
+  boards: PropTypes.arrayOf(boardPropTypes.board),
```

Comment:
Shouldn't this be required ? Otherwise [this call](https://github.com/sbalay/trello-redux-example/pull/2/files#diff-12b0262f4e9ffc58518a18d94f819ba0R11) could crash the app silently.